spec/classroom_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../classroom'
+require_relative '../person'
+require_relative '../student'
+
+describe Classroom do
+  before do
+    @classroom = Classroom.new('English')
+  end
+
+  describe '#initialize' do
+    it 'initializes with a label' do
+      expect(@classroom.label).to eq('English')
+    end
+  end
```

Comment:
- [ ] like the previous CR mentioned, in this project you are expected to test all methods in all classes which in this class should include the `add_student` method.
this test will look similar to the example I gave in the previous comment.

Reply:
@mikethreels we implemented the changes but since the project of prserve data we did it in my partners repo, the test of adding rentals and classroom here works but saves the added data and returns it in this format 
(#<Person:0x000055d34aef8cd8 @id=526, @name="Unknown", @age=20, @parent_permission=true, @rentals=[#<Double "rental">]>)

Reply:
@mikethreels thats the only issues

Reply:
@ksigei can you explain your issue a bit more because I'm not sure how to help you with this information.

Reply:
@mikethreels , on the previous project, we were supposed to preserver data using json. So we did it in my partners repo.
Now, in this repo, when you add a new data, it is save in a different format compared with the one the test requires, check this (#<Person:0x000055d34aef8cd8 @id=526, @name="Unknown", @Age=20, @parent_permission=true, https://github.com/rentals=[#<Double "rental">]>)

Reply:
@ksigei, in that case, I would advise you to change your code because your tests aren't passing due to that. please correct me if I understand you wrong.

Reply:
@mikethreels if we write tests on another repo now and create a new pull request, we will be in trouble. If we also start to refactor this code, there is no time as a lot is waiting for us, and reviewers take time to send feedback today.

Reply:
@ksigei I understand, let me know if you need anything else.

Reply:
@mikethreels you understand changing the code? or?

Reply:
@mikethreels you were not specific

Reply:
@ksigei my apologies I had to go offline but I see you already got your project approved. Congratulations 👏

Reply:
@mikethreels no problem, thanks